CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 - add [.editorconfig file](http://editorconfig.org/)
 - add (experimental/unsupported?) saltstack files
+- Publish expired DNS transactions
```

Comment:
Why do we have both Added and Improvements, anyway? On http://keepachangelog.com/ I don't see Improvements.